config/connect-streams-file-source.properties

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name=local-file-source
```

Comment:
This is the same as in `connect-file-source.properties`. If config and offset storage aren't changed, then they will conflict.

Is there any reason we need to duplicate all of this? These configs all mostly look the same (only the topic seems to have changed). It makes sense that they wouldn't really be different since there shouldn't be anything Streams specific to the connector anyway. If it helps make things a bit clearer, maybe just change the topic in `connect-file-source.properties` to be more generic and then reuse all the existing configs?